src/digital/v1_compat.rs

```diff
@@ -11,21 +11,26 @@ pub struct OldOutputPin<T> {
     pin: T,
 }
 
-impl <T, ERR> OldOutputPin<T>
+impl <T, E> OldOutputPin<T>
 where
-    T: v2::OutputPin<Error=ERR>,
-    ERR: core::fmt::Debug,
+    T: v2::OutputPin<Error=E>,
+    E: core::fmt::Debug,
 {
     /// Create a new OldOutputPin wrapper around a v2::OutputPin
     pub fn new(pin: T) -> Self {
         Self{pin}
     }
+
+    /// Fetch a reference to the inner v2::OutputPin impl
+    pub fn inner(&self) -> &T {
```

Comment:
Hmm, this is new. I see how this is necessary for the tests but could this be made private?
```suggestion
    fn inner(&self) -> &T {
```

Reply:
Hmm kindof, i've made it private and gated one with `#[cfg(test)]` to stop the compiler being sad about the lack of use outside of tests (and removed the unused one).